### DISCUSSION TITLE
ERROR: No README data found!

### DISCUSSION THREAD

**Discussion by superman66** [verdaccio/verdaccio#2362] [Q&A] [answered]
Hi, I found some package can't display README.md, and show "ERROR: No README data found!".
But when When I checked the released package, it did contain the readme `README.md`.
![image](https://user-images.githubusercontent.com/12592949/127424733-bec5e569-45e7-432a-9730-8e27cc3fa41c.png)

This is my environment:
 * **verdaccio version**: 5.1.2
 * **node version** : 16.5.0
 * **package manager**: npm@7
 * **os**: linux
 * **platform**: npm

I'm not sure if it's a bug or a setup error.
Thanks in advance!

**Comment by juanpicado** [comment, marked as answer]
could you share a example package what you are publishing?. The npm7 has changed the way README is being send to the registry.

**Reply by juanpicado** [reply]
ref #1905

**Reply by superman66** [reply]
Thank you for your reply. Suddenly README.md can be displayed again.
I'll look for specific reasons.
